LINK/etc/profile.d/evm.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+# note: this will override $PATH
```

Comment:
Will it?

Reply:
the properties file for `enable` (now called `production.properties`